js/src/css/shared/_gutenberg-components.scss

```diff
@@ -1,15 +1,12 @@
 // Import Gutenberg components that aren't already imported in the lowest WC Admin version we support
-@import "node_modules/@wordpress/components/src/button/style"; // required for tab-panel
+// @import "node_modules/@wordpress/components/src/button/style"; // required for tab-panel
```

Comment:
💅 

Why the comment instead of deletion? Shouldn't we just rely on GIT history for this?

Reply:
Thanks for catching this! 👍  It should not be deleted. Looks like it has impacts on this issue but it's not the cause, I accidentally added the change to git commit.

Reverted by https://github.com/woocommerce/google-listings-and-ads/pull/1219/commits/5f8195942cf9cfe11d03818e783766a5583b40c5.